extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -47,29 +47,45 @@ public function createHorizontalGroupData( $groupData ) {
 	}
 
 	/**
-	 * checks if infobox item is the title or title inside the hero module
-	 * and if so, removes from it all HTML tags.
+	 * check if infobox item is a title, title inside the hero module or a label
+	 * and if so, remove from it all HTML tags.
 	 *
 	 * @param string $type type of infobox item
 	 * @param array $data infobox item data
 	 * @return array infobox $data with sanitized title param if needed
 	 */
-	public function sanitizeInfoboxTitle( $type, $data ) {
-		if ( $type === 'title' && !empty( $data[ 'value' ] ) ) {
-			$data[ 'value' ] = trim( strip_tags( $data[ 'value' ] ) );
-
-			return $data;
-		}
-		if ( $type === 'hero-mobile' && !empty( $data[ 'title' ][ 'value' ] ) ) {
-			$data[ 'title' ][ 'value' ] = trim( strip_tags( $data[ 'title' ][ 'value' ] ) );
-
-			return $data;
+	public function sanitizeTitlesAndLabels( $type, $data ) {
```

Comment:
Would it make sense to call it sanitizeInfoboxFields since the description covers all cases? 'and' in method name is usually a sign of something twisted happening ;-)